CMR/python/cmr/auth/token.py

```diff
@@ -56,48 +55,56 @@ def token_literal(token_text):
     return lambda _ : token_text
 
 
-def token_config(options: dict = None):
+# document-it: {"key":"cmr.token.value", "default":"None"}
+def token_config(config: dict = None) -> str:
```

Comment:
remove space before dict

Reply:
I found a definitive answer to this question. Check out doc/style_rules.md. Going with this style because I found a good citation on what the rules should be.